content/concepts/tokens/nftoken-auctions.md

```diff
@@ -0,0 +1,60 @@
+---
+html: nftoken-auctions.html
+parent: non-fungible-tokens.html
+blurb: You can assign another account to mint NFTs in your stead.
+labels:
+ - Non-fungible Tokens, NFTs
+---
+
+# Running an NFT Auction
+
+There are several ways to run an auction, each with advantages and disadvantages.
+
+## Run the Auction Off the XRPL, Complete the Purchase on XRPL
+
+This flow is listed first as it is the most straightforward. You store your bids in a private database. You take a cut of the winning bid prior to sending the Buyer/Seller the XRPL transaction that completes the purchase. Note that because NFTokenOffer objects can always be canceled by their creator, it is not currently possible to implement a binding offer.
+
+## Run the Auction in Brokered Mode, with a Reserve
+
+Run the auction in brokered mode, as an auction with a reserve.
+
+![Auction in Brokered Mode with a Reserve](img/nft-auction1.png "Auction in Brokered Mode with a Reserve")
+
+1. The Seller creates the NFT, then sets the auction reserve price using NFTokenCreateOffer, specifying the broker account as the destination.
+1. Buyers make bids using NFTokenCreateOffer, setting the broker account as the destination.
+1. The Broker selects the winning bid, completes the sale using NFTokenAcceptOffer, collecting the broker fee. Then the Broker cancels any losing bids using NFTokenCancelOffer.
+
+**Pros:**
+
+- The entire auction happens on the XRPL, including your broker fee.
```

Comment:
Maybe I'm misreading the topic, but aren't all three methods listed taking place entirely on the XRPL?

Reply:
No, the first option handles the _auction_ independent of the XRPL, completing the sale as a normal transaction. Options 2 and 3 use the broker process to handle the auction using the brokered mode feature.